pkg/installer/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#
+# Zeninstaller
+#
+# Install Ze*ix distribution to a device.
+#
+# Usage:
+#
+#   install.sh <device>
+
+echo "STARTING INSTALLATION"
```

Comment:
Makes sense to add $1 to echo line (so one can see which disk will be clobbered)

Reply:
Good point.